website/source/docs/connect/configuration.html.md

```diff
@@ -32,117 +35,66 @@ You may also configure Consul to use an external
 [certificate management system](/docs/connect/ca.html), such as
 [Vault](https://vaultproject.io).
 
-No agent-wide configuration is necessary for non-server agents. Services
-and proxies may always register with Connect settings, but they will fail to
-retrieve or verify any TLS certificates. This causes all Connect-based
+Services and proxies may always register with Connect settings, but they will
+fail to retrieve or verify any TLS certificates. This causes all Connect-based
 connection attempts to fail until Connect is enabled on the server agents.
 
--> **Note:** Connect is enabled by default when running Consul in
-dev mode with `consul agent -dev`.
+Other optional Connect configurations that you can set in the server
+configuration file include:
+
+- [the options listed under `connect`](/docs/agent/options.html#connect)
+- [token replication](/docs/agent/options.html#acl_tokens_replication)
+- [dev mode](/docs/agent/options.html#_dev)
+- [server host name verification](/docs/agent/options.html#verify_server_hostname)
 
-~> **Security note:** Enabling Connect is enough to try the feature but doesn't
+If you would like to use Envoy as your Connect proxy you will need to [enable
+gRPC](/docs/agent/options.html#grpc_port). Additionally if you plan on using the
+observability features of connect, it can be convenient to enable [configuration
+entries](/docs/agent/options.html#config_entries).
```

Comment:
I think "enable" is the wrong verb here. I'm not quite sure how to word it but the recommendation should be in general to use config_entries, and one way of doing so is via the _server's_ config file. I'm a tiny bit hesitant to mention that only though - I think the more useful pattern in general is to manage those via the CLI/API (or eventually via terraform once we build support). The Config file approach is easier in some cases and is OK but I'd be wary of promoting it as the most visible/recommended path by putting it here?